ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -852,17 +851,54 @@ <h3 class="title">
 
   <div class="save-edits">
     <mat-divider *ngIf="isEditMode || isPkEditMode || isFkEditMode"></mat-divider>
+
     <div class="save-button" *ngIf="isEditMode && currentTabIndex === 0">
-      <button mat-raised-button color="primary" (click)="toggleEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveColumnTable()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isPkEditMode && currentTabIndex === 1">
-      <button mat-raised-button color="primary" (click)="togglePkEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="savePk()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="togglePkEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isFkEditMode && currentTabIndex === 2">
-      <button mat-raised-button color="primary" (click)="toggleFkEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveFk()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleFkEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
     </div>
     <div class="save-button" *ngIf="isIndexEditMode && currentTabIndex === -1">
-      <button mat-raised-button color="primary" (click)="toggleIndexEdit()">SAVE & CONVERT</button>
+      <button mat-raised-button color="primary" (click)="saveIndex()">SAVE & CONVERT</button>
+      <button
+        class="cancel-button"
+        mat-raised-button
+        color="primary"
+        (click)="toggleIndexEdit()"
+        *ngIf="currentObject!.isSpannerNode"
+      >
+        CANCEL
+      </button>
```

Comment:
On editing primary order based on suggestion to make the interleavable , getting the msg to go to interleave tab and convert to interleave but no interleave tab appears - https://drive.google.com/file/d/1tNmiMy3y8E1-XNi3ZHCHMCljYv6P6d_S/view?usp=sharing&resourcekey=0-3jAz7JRMQtlFqD4naku6Nw

Reply:
Can you take the latest pull and test if possible. Interleave tab is appearing for me.

Reply:
showing now, it was due to some issue related to primary key order edit i guess